### PR TITLE
Spelling Corrections

### PR DESCRIPTION
**Pull request by randbw**
Some spelling corrections picked up as I read through microsite and code. 

Hopefully not too enthusiastic

### CODE REVIEW

File: microsite/src/main/tut/usage/schedules.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Schedules allow you to define and compose flexible recurrence schedules, which c
    * `IO.retryOrElse` — Retries an action until it succeeds, with a fallback for errors.
    * `IO.retryOrElse0` — Retries an action until it succeeds, with a more powerful fallback for errors.
 
-Schedules define stateful, possibly effectful, recurring schedules of events, and compose in a variety of ways.
```
Comment (NeQuissimus) [inline microsite/src/main/tut/usage/schedules.md]: I think we can leave the Oxford comma :)

**Comment by randbw** [comment]
Now everyone's aware I don't know how to use it :O

File: microsite/src/main/tut/usage/schedules.md (hunk 1 above)
Comment (randbw) [reply microsite/src/main/tut/usage/schedules.md]: Secret is out. I don't know how to use it. I have reading to do

Comment (randbw) [reply microsite/src/main/tut/usage/schedules.md]: fixed :)

**Pull request merged**